common.proto

```diff
@@ -20,10 +20,12 @@ message FormField {
   string label = 2;
   bool required = 3;
   optional string description = 4;
+  optional string default_value = 5;
+  optional string placeholder = 6;
   oneof type {
-    TextField text_field = 5;
-    DropdownField dropdown_field = 6;
-    ToggleField toggle_field = 7;
+    TextField text_field = 7;
```

Comment:
Please don't renumber fields. It makes these diffs awful. If we want to we can do one big renumber at the end before we go GA.